File: 프로그래머스/1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.py
def solution(arr):
    answer = []
    num=arr[0]
    for i in arr :
        if i <num :
            num=i
    for j in arr :
        if j==num :
            continue
        else :
            answer.append(j)
    if not answer :
        answer.append(-1)
    return answer

File: 프로그래머스/1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.java
class Solution {
    public int[] solution(int[] arr) {
        int point = 0;
        int ans_point=0;
        
        if (arr.length == 1){
            return new int[]{-1};
        }
        int[] answer = new int[arr.length -1];
        
        for (int i = 0; i< arr.length; i++){
            if (arr[point] > arr[i]){
                point = i;
            }
        }
        
        for (int i =0; i<arr.length; i++){
            if( i != point){
                answer[ans_point] = arr[i];
                ans_point++;
            }
        }
        
        
        return answer;
    }
}